runtime/tests/interpreter/account_test.go

```diff
@@ -501,6 +503,56 @@ func TestInterpretAuthAccount_load(t *testing.T) {
 			require.Len(t, getAccountValues(), 1)
 		})
 	})
+
+	t.Run("inclusiverange with contains", func(t *testing.T) {
+
+		t.Parallel()
+
+		address := interpreter.NewUnmeteredAddressValueFromBytes([]byte{42})
+
+		inter, getAccountValues := testAccount(
+			t,
+			address,
+			true,
+			`
+				fun save() {
+					let ir = InclusiveRange(10, 20)
+					account.save(ir, to: /storage/ir)
+				}
+
+				fun loadIRAndCallContains(): InclusiveRange<Int>? {
+					let ir = account.load<InclusiveRange<Int>>(from: /storage/ir)
+					let containsFifteen = ir!.contains(15)
+					return ir
+				}
```

Comment:
I am facing a strange issue while adding a new runtime test to save and load inclusive range in two different transactions.

https://github.com/onflow/cadence/pull/2523/commits/dd77fb755cb101d46462a4d1b655013aa10daf3c -- The save of InclusiveRange in the first transaction fails with error

```
Execution failed:
        	            	error: failed to encode map value: cannot store non-storable value
        	            	--> 0100000000000000000000000000000000000000000000000000000000000000
```

Its strange because the tests I added in https://github.com/onflow/cadence/commit/a4d4c96d888de388dd735d96ae74c16148cb1c46 works. So not sure what the difference is between these two cases.

Reply:
Per discussion on Discord, we'll add support for storing InclusiveRange in account storage in a future PR. For now, I've added a test case which indicates that this is an unsupported feature.